src/sbvr-api/sbvr-utils.coffee

```diff
@@ -664,6 +705,10 @@ exports.getAbstractSqlModel = getAbstractSqlModel = (request) ->
 	request.abstractSqlModel ?= abstractSqlModels[request.vocabulary]
 	return request.abstractSqlModel
 
+exports.loadTranslations = (translations) ->
+	Promise.each translations, (translation) ->
+		translation.setup(exports, exports.translations)
```

Comment:
Still not sure what API we should expose to translations, went with this to simplify for the time being